swig/cpp/src/Session.hpp

```diff
@@ -70,20 +70,21 @@ class Session
     /** Wrapper for [sr_list_schemas](@ref sr_list_schemas) */
     S_Yang_Schemas list_schemas();
     /** Wrapper for [sr_get_schema](@ref sr_get_schema) */
-    std::string get_schema(const char *module_name, const char *revision,
-                           const char *submodule_name, sr_schema_format_t format);
+    std::string get_schema(const char * /* module_name */, const char* /* revision */,
```

Comment:
Please do not change the declarations like this. The name of the argument is an important part of the function prototype.